.github/workflows/test.yml

```diff
@@ -182,86 +182,6 @@ jobs:
         run: |
           pip cache remove llama_cpp_python
 
-  docs:
-    needs: ["test-workflow-ready"]
-    runs-on: ubuntu-latest
-    steps:
-      - name: "Harden Runner"
-        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0 # v2.12.0
-        with:
-          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
-
-      - name: Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          # https://github.com/actions/checkout/issues/249
-          fetch-depth: 0
-
-      - name: Install dependent PRs if needed
-        uses: depends-on/depends-on-action@61cb3f4a0e2c8ae4b90c9448dc57c7ba9ca24c35 # main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Setup Python 3.11
-        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
-        with:
-          python-version: 3.11
-          cache: pip
-          cache-dependency-path: |
-            **/pyproject.toml
-            **/requirements*.txt
-
-      - name: Remove llama-cpp-python from cache
-        run: |
-          pip cache remove llama_cpp_python
-
-      - name: Run tox docs target (expect failure since tox is not present)
-        run: |
-          make docs
-        continue-on-error: true
-
-      - name: Run tox docs target
-        id: tox-docs
-        run: |
-          make man || echo "status=failed" >> "$GITHUB_OUTPUT"
-        continue-on-error: true
-
-      - name: Check for 'make man' failure
-        run: |
-          if [[ "${{ steps.tox-docs.outputs.status }}" != "failed" ]]; then
-            echo "'make man' did not fail as expected"
-            exit 1
-          fi
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install tox
-
-      - name: Run tox docs target
-        run: |
-          make docs
-
-      - name: Check that man pages were generated
-        uses: andstor/file-existence-action@076e0072799f4942c8bc574a82233e1e4d13e9d6 # v3.0.0
-        with:
-          files: 'man/*.1'
-          fail: true
```

Comment:
even if we don't deploy, shouldn't we at least check that `make man` does something reasonable?